configuration/log-config-0.yaml

```diff
@@ -62,23 +62,27 @@ options:
       # Change the `subtrace` value to `Neutral` in order to log
       # `UTxO`-related messages during epoch validation.
       subtrace: NoTrace
-    '#messagecounters.aggregation':
+    'cardano.#messagecounters.aggregation':
```

Comment:
Just a stray thought, not really critical -- do we really need this `cardano` prefix everywhere?

It seems that everything in IOHK is about Cardano, somehow, so it seems.. implicit? : -)

Reply:
at the moment, yes. there will be changes in the framework on the context naming ... (to be expected)